### PR TITLE
[IMP] calendar: add "Organizer" field to .ics data

### PR DESCRIPTION
**Pull request by flch-odoo**
If enough information is available on the meeting, this commit will prevent
undesirable "Unknown Organizer" notices from .ics imports.

Task-2743866

### CODE REVIEW

**Review by awa-odoo — COMMENTED** [review]
Hello @flch-odoo 

Here is a quick validation, thank you for your reactivity on this one.

Cheers!

**Comment by flch-odoo** [comment]
@awa-odoo , indeed much cleaner. Thank you!

File: addons/calendar/models/calendar_event.py [hunk 1]
```diff
@@ -1040,6 +1040,14 @@ def ics_datetime(idate, allday=False):
             for attendee in meeting.attendee_ids:
                 attendee_add = event.add('attendee')
                 attendee_add.value = u'MAILTO:' + (attendee.email or u'')
+
+            # Add "organizer" field if email available
+            if meeting.partner_id and meeting.partner_id.email:
```
Comment (tde-banana-odoo) [inline addons/calendar/models/calendar_event.py:1045]: Actually doing "if meeting.partner_id.email" is sufficient, as partner_id will be a void recordset and email will be Falsy.

**Review by tde-banana-odoo — COMMENTED** [review]
Review ! LGTM, one small comment :)

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**